pvr.argustv/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.argustv"
-  version="19.1.1"
+  version="19.1.2"
```

Comment:
Sorry, I spoke too soon. As you are adding functionality the version number should `19.2.0`. You would use `19.1.2` if you had fixed a bug.